web/api/netdata-swagger.json

```diff
@@ -839,6 +839,36 @@
         }
       }
     },
+    "/alarms_values": {
+      "get": {
+        "summary": "Get a list of active or raised alarms on the server",
```

Comment:
Should the "active" parameter be in the "parameters" section similarly to "all"?

Reply:
I agree, I updated the description. Thank you!

Reply:
@thiagoftsm I think @mfundul meant in the parameter section not in the description

something like 
```
{
            "name": "active",
            "in": "query",
            "description": "If passed, the raised alarms in state WARNING or CRITICAL are returned.",
            "required": false,
            "allowEmptyValue": true
}
```
but I see that it doesn't exist for the `/alarms` either .. :man_shrugging:

Reply:
I understood better what you said, I decided to update the text description instead to create another section thanks the conclusion you already arrived, it was never described for `/alarms`, but I copy the description that you gave us for both.
Thank you!

Reply:
I also changed the `yaml` file.